auth/src/main/java/org/openedx/auth/presentation/signin/SignInFragment.kt

```diff
@@ -76,6 +86,20 @@ class SignInFragment : Fragment() {
                     onForgotPasswordClick = {
                         viewModel.forgotPasswordClickedEvent()
                         router.navigateToRestorePassword(parentFragmentManager)
+                    },
+                    onLoginClickOauth = {
+                        val uri = Uri.parse("${BuildConfig.BASE_URL}oauth2/authorize")
+                            .buildUpon()
+                            .appendQueryParameter("client_id", BuildConfig.CLIENT_ID)
+                            .appendQueryParameter("redirect_uri", "${context.packageName}://oauth2Callback")
+                            .appendQueryParameter("response_type", "code")
+                            .build()
+                        val intent = CustomTabsIntent.Builder()
+                            .setUrlBarHidingEnabled(true)
+                            .setShowTitle(true)
+                            .build()
+                        intent.launchUrl(context, uri)
```

Comment:
Using the custom tab mechanism allows any browser to handle this AFAIK and that can use the users saved password etc.